lib/db_utils.go

```diff
@@ -233,10 +233,21 @@ var (
 	// 		<prefix, OwnerPublicKey [33]byte> -> <>
 	_PrefixAuthorizeDerivedKey = []byte{54}
 
+	// Prefix for Messaging Keys:
+	//		<prefix, PublicKey [33]byte, KeyName [32]byte> -> <>
+	_PrefixMessagingKey = []byte{57}
+
+	// Prefix for Message Recipients:
+	//		<prefix, PublicKey [33]byte, KeyIndex [10]byte> -> <>
+	_PrefixMessageRecipient = []byte{58}
+
+	// Prefix for Message to Message Party. Prefix works similarly to regular private messages.
+	_PrefixMessageToMessageParty = []byte{59}
+
 	// TODO: This process is a bit error-prone. We should come up with a test or
 	// something to at least catch cases where people have two prefixes with the
 	// same ID.
-	// NEXT_TAG: 55
+	// NEXT_TAG: 58
```

Comment:
this should be 60 based on your values - but let's merge in main so we're confident that we're not overlapping.